app/__init__.py

```diff
@@ -19,7 +19,9 @@
 compress = Compress()
 # Set up Flask-Login
 login_manager = LoginManager()
-login_manager.session_protection = 'strong'
+# TODO: Ideally this should be strong, but that led to bugs. Once this is
+# fixed, switch protection mode back to 'strong'
+login_manager.session_protection = 'basic'
```

Comment:
Session protection works by generating a key-value pair of a random hashed string on the user end and in the SESSION variable. If these are not equal, in `basic` mode, only routes which require a `fresh` session (i.e. not a remember_me session for example) will ask the user to reauthenticate. But we don't implement a check from `fresh` login on our routes. In `strong` mode, the user is forced to reauthenticate when the SESSION and user hashed string do not match _no matter what_. Hence it is best to remain at `strong` session_protection level.